themes/scss/widgets/buttons.scss

```diff
@@ -13,7 +10,7 @@
   line-height: 40px;
 }
 .CDB-Widget-link {
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores